docs/architecture/adr-031-msg-service.md

```diff
@@ -0,0 +1,238 @@
+# ADR 031: Protobuf Msg Services
+
+## Changelog
+
+- 2020-10-05: Initial Draft
+
+## Status
+
+Proposed
```

Comment:
```suggestion
Accepted
```

What does it take to make this ADR accepted? I'm in favor of implementing this ASAP, as a clean base for upcoming modules development.

Reply:
We don't actually have a process around ADR status. Let's discuss on today's architecture call.

Reply:
I would prefer to have it iteratively. We can speedup if needed. Usually a second round for something important is good, and will reach to the wider audience. It might be as simple as changing a status though. 

Also, for some cases it will be good to firstly prepare an example implementation / reference. While submitting that example, we can pair it with ADR status update PR. Sometimes it could be useful, because during the implementation we may find that something doesn't work or will require some updates.